os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -206,6 +208,13 @@ static void rtl8730e_control_backlight(uint8_t level)
 	lcdvdbg("level :%d , pwm level:%f\n", level, pwm_level);
 #if defined(CONFIG_LCD_ST7785) || defined(CONFIG_LCD_ST7701SN)
 	pwmout_write(&g_rtl8730e_config_dev_s.pwm_led, 1.0-pwm_level);
+	if (level == 0 && !(lcdc_irq_info.disable_isr)) {
```

Comment:
Can you check disable_isr state in rtl8730e_lcd_put_area()?

Reply:
Hi @ewoodev, i have tried adding below code, it is able to run fps_test when control backlight 0. 
        if (lcdc_irq_info.disable_isr) {
		InterruptEn(lcdc_irq_info.num, lcdc_irq_info.priority);
		lcdc_irq_info.disable_isr = 0;
	}

Reply:
Missed your edited comment, so do you want to return, or re-enable the isr?

Reply:
I think, return is better. 

Controlling IRQ in both places is at risk of collision.
In addition, backlight is expected to be controlled in the intentional situation of the app.